mtp_api/apps/credit/views.py

```diff
@@ -79,8 +79,8 @@ def get_field_filter(field):
                         return models.Q(**{'%s__startswith' % field: amount})
                 elif field == 'sender_name':
                     return (
-                        models.Q(**{'transaction__sender_name__icontains': word})
-                        | models.Q(**{'payment__cardholder_name__icontains': word})
+                        models.Q(**{'transaction__sender_name__icontains': word}) |
+                        models.Q(**{'payment__cardholder_name__icontains': word})
```

Comment:
Actually maybe we can keep the previous version. I think people suggest having the operators at the beginning of the line so that it's easier to scan instead of scrolling to the end of the lines with different lengths.